README.md

```diff
@@ -29,7 +33,12 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
 
 ## What you need
 
-You need to submit Github Token to access Github API.
+You need to submit Github Token to access Github API. Do this by generating a personal token listed in your own settings at: https://github.com/settings/tokens
+
+## How to use
+1) Add Chrome Extension
+2) Granted Curiosity your own github token
+3) Open up a new Chrome browser tab, you'll see a feed of Github projects which are being used heavily (starred). 
```

Comment:
I would reword it as: Open a new browser tab and run the `index.html` file in the root folder